packages/components/src/listbox/src/Listbox.css

```diff
@@ -144,7 +143,7 @@ IMPORTANT: The Listbox component hardcoded a few CSS values to enable dynamic sc
 
 /* OPTION | ICONS | END ICON */
 .o-ui-listbox-option-end-icon {
-    margin-left: calc(var(--o-ui-sp-1) + 0.125rem);
+    margin-left: calc(var(--o-ui-sp-2) + 0.125rem);
```

Comment:
Values not on scale values are a smell ;)

Reply:
The 0.125rem could be removed here actually and would not make a huge difference, I considered this one not a smell since its not on the vertical rhythm scale. It was to compensate the fact that icons are now 20x20 missing 2px each side, that spacing was included in the icon before.